Dockerfile

```diff
@@ -12,19 +12,7 @@ LABEL org.opencontainers.image.ref.name="jnovack/autossh" \
       org.opencontainers.image.revision=$COMMIT \
       org.opencontainers.image.url="https://hub.docker.com/r/jnovack/autossh/"
 
-ENTRYPOINT ["/entrypoint.sh"]
-ADD /entrypoint.sh /entrypoint.sh
-RUN chmod 755 /entrypoint.sh
+RUN apk --no-cache add autossh openssh-client
+COPY /entrypoint.sh /init
```

Comment:
This is not an `init` program. It does not [ensures the usual responsibilities of an init system, such as reaping zombie processes](https://docs.docker.com/engine/reference/run/#specify-an-init-process).

Please revert this change.